src/itkOMEZarrNGFFImageIO.cxx

```diff
@@ -634,23 +653,11 @@ OMEZarrNGFFImageIO::Read(void * buffer)
               << storeIORegion;
   }
 
-  if (false)
-  {
-  }
-  READ_ELEMENT_IF(float)
-  READ_ELEMENT_IF(int8_t)
-  READ_ELEMENT_IF(uint8_t)
-  READ_ELEMENT_IF(int16_t)
-  READ_ELEMENT_IF(uint16_t)
-  READ_ELEMENT_IF(int32_t)
-  READ_ELEMENT_IF(uint32_t)
-  READ_ELEMENT_IF(int64_t)
-  READ_ELEMENT_IF(uint64_t)
-  READ_ELEMENT_IF(float)
-  READ_ELEMENT_IF(double)
-  else
+  if (const IOComponentEnum componentType{ this->GetComponentType() };
+      !TryToReadFromStore<float, int8_t, uint8_t, int16_t, uint16_t, int32_t, uint32_t, int64_t, uint64_t, double>(
+        componentType, store, storeIORegion, buffer))
```

Comment:
- @tbirdso, @dzenanz pull request #35 commit c9cec9f6503b18b848e327960870b5b4edd6170c (July 31, 2023) placed an extra `READ_ELEMENT_IF(float)` call in front of the other `READ_ELEMENT_IF` calls, do you remember if that was intentional?

Reply:
Hi @N-Dekker , sorry, I don't recall a specific reason. I think it was probably a stray leftover from my testing and could be removed.

Reply:
I don't remember, but this could be a rebasing artifact or something similar.

Reply:
Thanks to you both! Resolved by commit "STYLE: Remove duplicate `READ_ELEMENT_IF(float)` call from Read(void *)"